tests/testthat/test-plot.vp.R

```diff
@@ -1,43 +1,21 @@
 test_that("plot.vp() returns error on incorrect parameters", {
+  # use plot.vp() to avoid defaulting to base plot()
   expect_error(plot.vp("not_a_vp"))
-  expect_error(plot.vp(example_vp, quantity = "not_a_quantity"))
+  expect_error(plot(example_vp, quantity = "not_a_quantity"))
 
   # Test error on "param" instead of "quantity"
-  expect_error(plot.vp(example_vp, param = "ff"))
+  expect_error(plot(example_vp, param = "ff"))
 
   # Test warning for deprecated arguments
```

Comment:
I think these are best bundled in a separate tests, called "plot.vp() warns for deprecated arguments"